src/core/reopt.jl

```diff
@@ -213,8 +213,12 @@ function build_reopt!(m::JuMP.AbstractModel, p::REoptInputs)
 				@constraint(m, [ts in p.time_steps], m[:dvGridToStorage][b, ts] == 0)
 				@constraint(m, [t in p.techs.elec, ts in p.time_steps_with_grid],
 						m[:dvProductionToStorage][b, t, ts] == 0)
+				@constraint(m, [ts in p.time_steps], m[Symbol("dvStorageToGrid")][ts] == 0)  # if there isn't a battery, then the battery can't export power to the grid
 			end
 		else
+			# Additional variables for exporting storage energy to the grid
+			m[:binBattCharging] = @variable(m, p.time_steps], base_name=dv, Bin) # Binary for battery charging (vs discharging)
```

Comment:
is there a reason you didn't put this where the other battery vars are defined?

Reply:
this also should only be defined if there for elements in storage.types.elec not storage.types.all

Reply:
Well, if it has "batt" in its name and is only used in the elec storage constraints, it should. Or, the it should be generalized to all storage.